doc/whats_new.rst

```diff
@@ -58,6 +58,9 @@ Changelog
 - Added feature to read/write :class:`~hnn_core.Network` configurations to
   json, by `George Dang`_ and `Rajat Partani`_ in :gh:`757`
 
+- Added :class:`~hnn_core/viz/NetworkPlotter` to visualize and animate network simulations,
+  by `Nick Tolley`_ in :gh:`649`
```

Comment:
```suggestion
  by `Nick Tolley`_ in :gh:`649`.
```
I know we haven't been good about this so far, but we should try to be more consisent with punctuation.